tonic/utils/swag/helpers.go

```diff
@@ -6,12 +6,19 @@ import (
 	"regexp"
 	"strings"
 
+	"github.com/loopfz/gadgeto/tonic"
 	"github.com/loopfz/gadgeto/tonic/utils/swag/swagger"
 )
 
 const (
-	query_tag = "query"
-	path_tag  = "path"
+	queryTag         = tonic.TagQuery
+	pathTag          = tonic.TagPath
+	validateTag      = tonic.TagValidation
+	defaultTag       = "default"
```

Comment:
This is declared in `tonic`, maybe we should reuse it. `fiizz` will use this exported tag name from `tonic` to keep sync.

Reply:
Done, with some others. https://github.com/loopfz/gadgeto/pull/38/commits/e7868d461f68643decb52d53a0c07084ecd2fa2c